tests/unittests/tests-libcose/tests-libcose.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  unittests
+ * @{
+ *
+ * @file
+ * @brief       Unittests for the libcose package
+ *
+ */
+#ifndef TESTS_LIBCOSE_H
+#define TESTS_LIBCOSE_H
+
+#include "embUnit/embUnit.h"
+#include "random.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief MANDATORY function for collecting random Bytes
+ *        required by the tweetnacl package
+ */
+extern void randombytes(uint8_t *target, uint64_t n);
```

Comment:
Hmm... I think this should be embedded in RIOT, but AFAIR there was a lot of discussion which ended into "prefer not having one instead of a non-truly-super-well-seeded-without-flaws random function"... But well, anyways it's not clear that the function is ALSO implemented in the tweetnacl package.

Reply:
Removed the declaration here as it was indeed not needed, already taken care of by the tweetnacl package.